MIGRATION.md

```diff
@@ -70,6 +71,38 @@ Tip: Provide the `--transform`/`-t` argument at the end of the command, so that
 > ./node_modules/.bin/circuit-ui migrate -l JavaScript -l TypeScript -t codemod-name
 > ```
 
+## From v4 to v5
+
```

Comment:
Can we add a note here about the v5 not being out yet, but adding that the draft migration guide is a good way to handle deprecations early?